suite-common/suite-sync-types/src/turnOnSuteSync.ts

```diff
@@ -1,10 +1,13 @@
 import { Dispatch } from '@reduxjs/toolkit';
 
+import { TurnOnSuiteSyncForWallet } from './storage/turnOnSuiteSyncForWallet';
+
 export type TurnOnSuiteSync = () => void;
 
 export type TurnOnSuiteSyncDep = { turnOnSuiteSync: TurnOnSuiteSync };
 
 export type CreateTurnOnSuiteSyncDeps = {
     getState: () => any;
     dispatch: Dispatch;
+    turnOnSuiteSyncForWallet: TurnOnSuiteSyncForWallet;
```

Comment:
```suggestion
    & TurnOnSuiteSyncForWalletDep
```

This saves you from repeating the `turnOnSuiteSyncForWallet` key and also enforces same key in all places where you depend on.